### PR TITLE
[MX-424] Partner page design refresh

### PR DESCRIPTION
**Pull request by ashfurrow**
The type of this PR is: Enhancement ✨ 

This PR resolves: **[MX-424]**

### Description

This migrates the partner page to the new typography. I took cues from #3412 to refactor things to be a bit cleaner. Probably best to review this PR ignoring whitespace. I also migrated the unit tests from Enzyme to ReactTestRenderer 💆‍♂️ 

### Test Plan

We'll get @samchieng to give it a look in the beta.

### Screenshots

| Before | After |
|---|---|
| Screen Shot 2020-07-16 at 5 26 53 PM  | Screen Shot 2020-07-17 at 11 27 10 AM |
| Screen Shot 2020-07-16 at 5 26 48 PM  |Screen Shot 2020-07-17 at 11 26 58 AM  |

[MX-424]: https://artsyproduct.atlassian.net/browse/MX-424

### CODE REVIEW

**Comment by samchieng** [comment]
Nice! ✨ A few comments:

### **Header**
- The current artist page header truncates the # of followers and # of works (i.e. 2.6k works • 137.5k followers) – is this something we can replicate here?
- The current artist page header also uses a smaller `follow` button, that's right-aligned – can we also duplicate on the partner page for consistency?

Artist page below for reference:

> image

### **Overview Tab**
- Change the Locations text from `Sans 3` to `Sans3t` (tighter line-height for this!)

### **[NEW] Section Headers**
- Last thing that I completely missed and didn't add in the original ticket is updating the section headers to match the new styles 🤦🏻‍♀️. We changed these recently on the artist page, so instead of the `Sans Medium 3` that we see on the Partner pages, we updated to `Sans Regular 4t.` This includes 3 section header changes below:

1. "**Artists (#)**" on the `Overview Tab`

> image

2. "**Current and upcoming shows**" on the `Shows Tab`
3. "**Past shows**" on the `Shows Tab`

> image

**Comment by ashfurrow** [comment]
Awesome, thanks Sam! Yeah we can definitely make those changes. I'll follow up in this PR 👍

**Comment by ashfurrow** [comment]
@samchieng Okay! I've applied that feedback and updated the screenshots in the pull request description 👍

**Comment by samchieng** [comment]
Awesome!! Last tweak is the section headers should be in `Regular` instead of `Medium`. Then lets ship!! 🚢

File: src/lib/utils/__tests__/formatText-tests.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { formatText } from "../formatText"
+
+describe(formatText, () => {
```
Comment (brainbicycle) [inline src/lib/utils/__tests__/formatText-tests.ts:3]: Nice test!

**Pull request merged**